python/doc/examples/graphs/plot_graphs_contour.py

```diff
@@ -0,0 +1,110 @@
+"""
+A quick start guide to contours
+===============================
+"""
+# sphinx_gallery_thumbnail_number = 6
+# %%
+#
+# In this example we show how to create contour graphs and how to make the most of their display settings.
+
+# %%
+# The `draw` method, the `Graph` and `Contour` classes
+# ----------------------------------------------------
+#
+# The simplest way to create a contour graph is to use the `draw` method of a bidimensional function.
+
+# %%
+import openturns as ot
+import openturns.viewer as viewer
+
+# %%
+# We build a bidimensional function (function of x and y), define the study domain and the sample size
+
+# %%
+f = ot . SymbolicFunction(['x', 'y'], ['exp(-sin(cos(y)^2*x^2+sin(x)^2*y^2))'])
+XMin = -5.
+XMax = 5.
+YMin = -5.
+YMax = 5.
+NX = 75
+NY = 75
+
+# %%
+# We build the graph by calling the `draw` method and display it
+
+# %%
+graph = f.draw([YMin, YMin], [XMax, YMax], [NX, NY])
+view = viewer.View(graph)
+
+# %%
+# The graph contains an unique drawable whose implementation is of class `Contour`
+
+# %%
+contour = graph.getDrawable(0).getImplementation()
+print(type(contour).__name__)
+
+# %%
+# Another way to build the contour is to build the data sample and give it to the constructor of the `Contour` class
+
+# %%
+inputData = ot.Box([NX, NY]).generate()
+inputData *= [XMax - XMin, YMax - YMin]
+inputData += [XMin, YMin]
+data = f(inputData)
+
+contour = ot.Contour(ot.Box([NX]).generate() * [XMax - XMin] + [XMin], ot.Box([NY]).generate() * [YMax - YMin] + [YMin], data)
```

Comment:
It seems to me that this line is too long. I suggest to use [black](https://github.com/psf/black).

Reply:
I use flake8 configured as in the lint.sh script.
The maximum length of lines is 185

Reply:
flake8 is to ensure the style of the code, not the compliance with PEP8. In the PEP8, the maximum line length is 79:

https://peps.python.org/pep-0008/#maximum-line-length

So 185 is _way_ too long.